projects/angular/src/accordion/_accordion.clarity.scss

```diff
@@ -460,105 +251,38 @@
     }
 
     .clr-accordion-panel:first-child .clr-accordion-header {
-      @include mixins.css-var(
-        border-top-left-radius,
-        clr-accordion-border-radius,
-        accordion-variables.$clr-accordion-border-radius,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-top-right-radius,
-        clr-accordion-border-radius,
-        accordion-variables.$clr-accordion-border-radius,
-        variables.$clr-use-custom-properties
-      );
+      border-top-left-radius: accordion-variables.$clr-accordion-border-radius;
+      border-top-right-radius: accordion-variables.$clr-accordion-border-radius;
     }
 
     .clr-accordion-panel:last-child {
       .clr-accordion-content,
       .clr-accordion-header {
-        @include mixins.css-var(
-          border-bottom-left-radius,
-          clr-accordion-border-radius,
-          accordion-variables.$clr-accordion-border-radius,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          border-bottom-right-radius,
-          clr-accordion-border-radius,
-          accordion-variables.$clr-accordion-border-radius,
-          variables.$clr-use-custom-properties
-        );
+        border-bottom-left-radius: accordion-variables.$clr-accordion-border-radius;
+        border-bottom-right-radius: accordion-variables.$clr-accordion-border-radius;
       }
     }
 
-    .clr-accordion-title {
-      @include mixins.css-var(font-weight, clr-accordion-title-font-weight, 500, variables.$clr-use-custom-properties);
-      @include mixins.css-var(
-        font-size,
-        clr-accordion-title-font-size,
-        variables.$clr_baselineRem_0_5416,
-        variables.$clr-use-custom-properties
-      );
-    }
-
```

Comment:
Why removing the individual styling and apply the typography mixin to the whole header?